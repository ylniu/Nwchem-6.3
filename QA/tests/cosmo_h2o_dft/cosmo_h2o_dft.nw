echo
start cosmo_h2o_dft
title "cosmo_h2o_dft"

geometry nocenter noautosym noautoz
  H    0.290  -1.478   2.133
  O   -0.526  -1.642   2.602
  H   -0.729  -0.989   3.278
end

basis noprint
  * library 6-31G
end


dft
  print medium
  xc hfexch
  iterations 200
  noio
  direct
  GRID nodisk
end

cosmo
  minbem 2
  maxbem 3
end

property
  efield
end

set dft:xcreplicated f
set fock:replicated .false.

task dft property
