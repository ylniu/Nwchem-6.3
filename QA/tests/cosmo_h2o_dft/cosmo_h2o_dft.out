All connections between all procs tested: SUCCESS
 argument  1 = cosmo_h2o_dft.nw



============================== echo of input deck ==============================
echo
start cosmo_h2o_dft
title "cosmo_h2o_dft"

geometry nocenter noautosym noautoz
  H    0.290  -1.478   2.133
  O   -0.526  -1.642   2.602
  H   -0.729  -0.989   3.278
end

basis noprint
  * library 6-31G
end


dft
  print medium
  xc hfexch
  iterations 200
  noio
  direct
  GRID nodisk
end

cosmo
  minbem 2
  maxbem 3
end

property
  efield
end

set dft:xcreplicated f
set fock:replicated .false.

task dft property
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.3
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2012
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = arcen
    program         = /home/d3y133/nwchem-releases/nwchem-6.3/QA/../bin/LINUX64/nwchem
    date            = Wed Mar 20 15:01:42 2013

    compiled        = Wed_Mar_20_14:44:54_2013
    source          = /home/d3y133/nwchem-releases/nwchem-6.3
    nwchem branch   = 6.3
    nwchem revision = 23861
    ga revision     = 10285
    input           = cosmo_h2o_dft.nw
    prefix          = cosmo_h2o_dft.
    data base       = /home/d3y133/nwchem-releases/nwchem-6.3/QA/scratchdir/cosmo_h2o_dft.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107201 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428802 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = /home/d3y133/nwchem-releases/nwchem-6.3/QA/scratchdir
  0 scratch   = /home/d3y133/nwchem-releases/nwchem-6.3/QA/scratchdir




                                NWChem Input Module
                                -------------------


                                   cosmo_h2o_dft
                                   -------------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 H                    1.0000     0.29000000    -1.47800000     2.13300000
    2 O                    8.0000    -0.52600000    -1.64200000     2.60200000
    3 H                    1.0000    -0.72900000    -0.98900000     3.27800000

      Atomic Mass 
      ----------- 

      H                  1.007825
      O                 15.994910


 Effective nuclear repulsion energy (a.u.)       9.1628016539

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
    -8.7815566689   -29.4853945998    49.5618435025


            XYZ format geometry
            -------------------
     3
 geometry
 H                     0.29000000    -1.47800000     2.13300000
 O                    -0.52600000    -1.64200000     2.60200000
 H                    -0.72900000    -0.98900000     3.27800000

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 O                |   1 H                |     1.80537  |     0.95536
    3 H                |   2 O                |     1.81708  |     0.96156
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    1 H                |   2 O                |   3 H                |   114.13
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/d3y133/nwchem-releases/nwchem-6.3/QA/../src/basis/libraries/>
  
  xc_inp: hfexch multiplicative factor not found.
                              NWChem Property Module
                              ----------------------


                                   cosmo_h2o_dft

  itol2e modified to match energy
  convergence criterion.

                                 NWChem DFT Module
                                 -----------------


                                   cosmo_h2o_dft



          ---------------
          -cosmo- solvent
          ---------------
 dielectric constant -eps-  =  78.40
 charge screening approach  =   1
 screen = (eps-1)/(eps    ) =   0.98724
 -lineq- algorithm          =   1
 -bem- low  level           =   2
 -bem- high level           =   3
 -bem- from -octahedral-
 solvent radius (ang.)      =   0.500
 atomic radii = 
 --------------
    1  1.000  1.300
    2  8.000  1.720
    3  1.000  1.300

 solvent accessible surface
 --------------------------

 ---------- ATOMIC COORDINATES (A.U.) ------------ VDWR(ANG.) --
     1    0.54802054   -2.79301501    4.03078553     1.300
     2   -0.99399587   -3.10293007    4.91706702     1.720
     3   -1.37761025   -1.86893900    6.19452179     1.300
 number of segments per atom =         32
 number of   points per atom =        128
 atom (   nspa,  nppa )
 ----------------------
    1 (     17,    50 )      50
    2 (     25,    80 )      80
    3 (     16,    48 )      48
 number of -cosmo- surface points =       58
 molecular surface =     39.495 angstrom**2
 molecular volume  =     20.367 angstrom**3
 G(cav/disp)       =      1.057 kcal/mol
 ...... end of -cosmo- initialization ......


  Caching 1-el integrals 
  itol2e modified to match energy
  convergence criterion.

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     3
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations: 200
          This is a Direct SCF calculation.
          AO basis - number of functions:    13
                     number of shells:     9
          Convergence on energy requested: 1.00D-07
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters        200 iters           200 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-11
          AO Gaussian exp screening on grid/accAOfunc:  16
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -75.75081731

      Non-variational initial energy
      ------------------------------

 Total energy =     -75.908302
 1-e energy   =    -121.677027
 2-e energy   =      36.605923
 HOMO         =      -0.468658
 LUMO         =       0.113391

   Time after variat. SCF:      0.0
   Time prior to 1st pass:      0.0

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       13.11            13105773
          Stack Space remaining (MW):       13.11            13107011

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
     COSMO gas phase
 d= 0,ls=0.0,diis     1    -75.9504799104 -8.51D+01  2.93D-02  3.18D-01     0.0
 d= 0,ls=0.0,diis     2    -75.9768470171 -2.64D-02  1.00D-02  9.03D-02     0.1
 d= 0,ls=0.0,diis     3    -75.9847782865 -7.93D-03  1.82D-03  9.54D-04     0.1
 d= 0,ls=0.0,diis     4    -75.9849420290 -1.64D-04  3.64D-04  2.14D-05     0.1
 d= 0,ls=0.0,diis     5    -75.9849470684 -5.04D-06  6.65D-05  4.82D-07     0.1
  Resetting Diis
 d= 0,ls=0.0,diis     6    -75.9849472001 -1.32D-07  6.19D-06  4.10D-09     0.1
 d= 0,ls=0.0,diis     7    -75.9849472013 -1.20D-09  1.24D-06  1.87D-10     0.1

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       13.11            13105517
          Stack Space remaining (MW):       13.11            13107011

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
     COSMO solvation phase
 d= 0,ls=0.0,diis     1    -75.9994249283 -1.45D-02  5.77D-03  4.09D-03     0.1
 d= 0,ls=0.0,diis     2    -76.0005158627 -1.09D-03  2.35D-03  8.33D-04     0.1
 d= 0,ls=0.0,diis     3    -76.0006865125 -1.71D-04  1.11D-03  1.45D-04     0.1
 d= 0,ls=0.0,diis     4    -76.0007195413 -3.30D-05  2.78D-04  4.52D-06     0.2
 d= 0,ls=0.0,diis     5    -76.0007210634 -1.52D-06  3.53D-05  1.03D-07     0.2
 d= 0,ls=0.0,diis     6    -76.0007210979 -3.45D-08  5.66D-06  2.76D-09     0.2


         Total DFT energy =      -76.000721097914
      One electron energy =     -123.129745765648
           Coulomb energy =       46.859607805545
    Exchange-Corr. energy =       -8.958765102801
 Nuclear repulsion energy =        9.162801653904

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.1s


                  COSMO solvation results
                  -----------------------
  
                 gas phase energy =       -75.9849472013
                 sol phase energy =       -76.0007210979
 (electrostatic) solvation energy =         0.0157738966 (    9.90 kcal/mol)

                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-2.055101D+01
              MO Center= -5.3D-01, -1.6D+00,  2.6D+00, r^2= 1.5D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.995763  2 O  s          

 Vector    2  Occ=2.000000D+00  E=-1.341516D+00
              MO Center= -4.3D-01, -1.5D+00,  2.6D+00, r^2= 5.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.493050  2 O  s                  4      0.471245  2 O  s          
     3     -0.214691  2 O  s          

 Vector    3  Occ=2.000000D+00  E=-7.208590D-01
              MO Center= -4.2D-01, -1.5D+00,  2.6D+00, r^2= 7.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.361176  2 O  pz                 5     -0.324365  2 O  px         
     1     -0.255804  1 H  s                 12      0.254128  3 H  s          
    11      0.202450  2 O  pz                 9     -0.181681  2 O  px         
     6      0.151904  2 O  py         

 Vector    4  Occ=2.000000D+00  E=-5.507668D-01
              MO Center= -5.9D-01, -1.7D+00,  2.6D+00, r^2= 6.6D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.444000  2 O  py                10      0.344066  2 O  py         
     5      0.333805  2 O  px                 9      0.258370  2 O  px         
     8     -0.255160  2 O  s          

 Vector    5  Occ=2.000000D+00  E=-4.998683D-01
              MO Center= -5.3D-01, -1.6D+00,  2.6D+00, r^2= 6.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.428616  2 O  pz                11      0.349636  2 O  pz         
     6     -0.344998  2 O  py                 5      0.315622  2 O  px         
    10     -0.281815  2 O  py                 9      0.257884  2 O  px         

 Vector    6  Occ=0.000000D+00  E= 2.279864D-01
              MO Center= -1.5D-01, -1.1D+00,  2.7D+00, r^2= 2.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.189198  2 O  s                 13     -1.003868  3 H  s          
     2     -0.988057  1 H  s                 10      0.341555  2 O  py         
     9      0.252391  2 O  px                 6      0.152935  2 O  py         

 Vector    7  Occ=0.000000D+00  E= 3.271292D-01
              MO Center= -1.2D-01, -1.1D+00,  2.7D+00, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.374557  1 H  s                 13     -1.349508  3 H  s          
    11      0.594114  2 O  pz                 9     -0.532705  2 O  px         
    10      0.250050  2 O  py                 7      0.235401  2 O  pz         
     5     -0.210685  2 O  px         

 Vector    8  Occ=0.000000D+00  E= 1.122348D+00
              MO Center= -5.1D-01, -1.5D+00,  2.7D+00, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      1.096462  3 H  s                  1     -0.909583  1 H  s          
    11     -0.603439  2 O  pz                 9      0.539580  2 O  px         
    13     -0.462695  3 H  s                  2      0.347950  1 H  s          
    10     -0.265374  2 O  py         

 Vector    9  Occ=0.000000D+00  E= 1.150442D+00
              MO Center= -1.2D-01, -1.2D+00,  2.6D+00, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      1.093106  1 H  s                 12      0.895367  3 H  s          
     2     -0.527719  1 H  s                 13     -0.478542  3 H  s          
     4     -0.278542  2 O  s                  9     -0.250528  2 O  px         
    10     -0.207691  2 O  py                 6     -0.194003  2 O  py         

 Vector   10  Occ=0.000000D+00  E= 1.161399D+00
              MO Center= -5.3D-01, -1.6D+00,  2.6D+00, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11     -0.699669  2 O  pz                 7      0.651125  2 O  pz         
    10      0.563601  2 O  py                 6     -0.527726  2 O  py         
     9     -0.510562  2 O  px                 5      0.480839  2 O  px         

 Vector   11  Occ=0.000000D+00  E= 1.188667D+00
              MO Center= -6.9D-01, -1.9D+00,  2.5D+00, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      0.882487  2 O  py                 6     -0.738782  2 O  py         
     9      0.659524  2 O  px                 5     -0.563024  2 O  px         
     4      0.262332  2 O  s                 11      0.228374  2 O  pz         
     7     -0.183388  2 O  pz                 2     -0.150981  1 H  s          

 Vector   12  Occ=0.000000D+00  E= 1.388164D+00
              MO Center= -5.0D-01, -1.6D+00,  2.6D+00, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.140945  2 O  pz                 9     -1.012772  2 O  px         
     2      0.927250  1 H  s                 13     -0.913258  3 H  s          
     7     -0.739580  2 O  pz                 5      0.659519  2 O  px         
    10      0.487876  2 O  py                 6     -0.313161  2 O  py         

 Vector   13  Occ=0.000000D+00  E= 1.674809D+00
              MO Center= -3.5D-01, -1.4D+00,  2.7D+00, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      2.719741  2 O  s                  4     -1.701343  2 O  s          
     2     -0.663654  1 H  s                 13     -0.648033  3 H  s          
    10      0.558932  2 O  py                 9      0.425501  2 O  px         
    12     -0.331639  3 H  s                  1     -0.327447  1 H  s          
     6     -0.156899  2 O  py         


 center of mass
 --------------
 x =  -0.92917465 y =  -3.01653702 z =   4.93895608

 moments of inertia (a.u.)
 ------------------
           3.924693981154           0.096308248243           1.896794114038
           0.096308248243           4.896728093959          -1.277822468465
           1.896794114038          -1.277822468465           3.966077698630

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -5.000000     -5.000000     10.000000

     1   1 0 0      0.627351      4.704454      4.704454     -8.781557
     1   0 1 0      0.833443     15.159419     15.159419    -29.485395
     1   0 0 1      0.209678    -24.676083    -24.676083     49.561844

     2   2 0 0     -5.147596     -7.624977     -7.624977     10.102359
     2   1 1 0     -2.796641    -14.257539    -14.257539     25.718437
     2   1 0 1      1.766913     23.595976     23.595976    -45.425038
     2   0 2 0     -9.627991    -48.973628    -48.973628     88.319266
     2   0 1 1      4.210508     74.552128     74.552128   -144.893749
     2   0 0 2     -1.887185   -124.963451   -124.963451    248.039717


          --------------
          Electric field
          --------------

 1 a.u. = 0.171524 10**(-8) dyn/esu 

   Atom       X         Y         Z                        Electric field (a.u.)
                                              X              Y              Z           Field
  ------------------------------------------------------------------------------------------------
    1 H    0.54802  -2.79302   4.03079       0.086192       0.011949      -0.053868       0.102341
    2 O   -0.99400  -3.10293   4.91707      -0.131730      -0.172801      -0.042274       0.221360
    3 H   -1.37761  -1.86894   6.19452      -0.024806       0.061308       0.068241       0.095030

 Task  times  cpu:        0.2s     wall:        0.4s
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  215      215     1091      259      598        0        0        0     
number of processes/call 1.90e+00 1.31e+00 1.63e+00 0.00e+00 0.00e+00
bytes total:             5.28e+05 1.63e+05 2.70e+05 0.00e+00 0.00e+00 0.00e+00
bytes remote:            1.31e+05 1.61e+04 8.44e+04 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 39368 bytes
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        26	        25
	current total bytes		         0	         0
	maximum total bytes		   3213664	  22510096
	maximum total K-bytes		      3214	     22511
	maximum total M-bytes		         4	        23


                                NWChem Input Module
                                -------------------





                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran Nair, J. Brabec, F. Aquino,
     S. Hirata, M. T. Hackler, K. Lopata, J. Mullin, P. Nichols, R. Peverati,
    J. Pittner, Y. Zhao, P.-D. Fan, R. J. Harrison, M. Dupuis, D. M. A. Smith,
 J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia
        L. Jensen, M. Swart, Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen,
      L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza,
   K. Hirao, R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
   X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann,
 G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang

 Total times  cpu:        0.2s     wall:        1.5s
